packages/terra-list/tests/wdio/selectable-list-spec.js

```diff
@@ -0,0 +1,139 @@
+/* global before, Terra, browser */
+
+describe('Selectable List', () => {
+  before(() => browser.setViewportSize(Terra.viewports('medium')[0]));
+
+  describe('Selectable List Chevron', () => {
+    before(() => browser.url('/#/raw/tests/terra-list/list/selectable-list/chevron-selectable-list'));
+
+    Terra.should.matchScreenshot();
+    Terra.should.beAccessible();
+  });
+
+  describe('Selectable List Focus', () => {
+    beforeEach(() => {
+      browser.url('/#/raw/tests/terra-list/list/selectable-list/default-selectable-list');
+      browser.waitForExist('ul');
+      browser.click('ul li:nth-child(1)');
+    });
+
+    Terra.should.themeCombinationOfCustomProperties({
+      testName: 'themed',
+      properties: {
+        '--terra-list-item-focus-background-color': '#fdebeb',
+        '--terra-list-item-focus-box-shadow': '5px 10px #dddddd',
+        '--terra-list-item-focus-outline': '1px dashed #000000',
+      },
+    });
+  });
+
+  describe('Selectable List Default', () => {
+    before(() => browser.url('/#/raw/tests/terra-list/list/selectable-list/default-selectable-list'));
+
+    Terra.should.matchScreenshot();
+
+    it('selects the first item', () => {
+      browser.keys(['Tab', 'Space']);
+    });
+    Terra.should.matchScreenshot('first item selected');
+
+    it('selects the second item', () => {
+      browser.keys(['Tab', 'Enter']);
+    });
+    Terra.should.matchScreenshot('second item selected');
+
+    it('selects the third item', () => {
+      browser.click('ul li:nth-child(3)');
+    });
+    Terra.should.matchScreenshot('third item selected');
+    Terra.should.beAccessible();
+  });
+
+  describe('Selectable List Disable Unselected', () => {
+    before(() => browser.url('/#/raw/tests/terra-list/list/selectable-list/disable-unselected-selectable-list'));
+
+    it('selects the first item', () => {
+      browser.keys(['Tab', 'Enter']);
+    });
+    Terra.should.matchScreenshot('first item selected');
+
+    it('selects the second item', () => {
+      browser.keys(['Tab', 'Enter']);
+    });
+    Terra.should.matchScreenshot('second item selected');
+
+    it('attempts to select the third item', () => {
+      browser.keys(['Tab', 'Enter']);
+    });
+    Terra.should.matchScreenshot('third item disabled');
+
+    Terra.should.beAccessible();
+  });
+
+  describe('Selectable List Items Divided', () => {
+    before(() => browser.url('/#/raw/tests/terra-list/list/selectable-list/items-divided-selectable-list'));
+
+    Terra.should.matchScreenshot();
+    Terra.should.beAccessible();
+  });
+
+  describe('Selectable List Non Selectable Item', () => {
+    before(() => browser.url('/#/raw/tests/terra-list/list/selectable-list/non-selectable-item-selectable-list'));
+
+    Terra.should.matchScreenshot();
+    it('attempts to select non-selectable item', () => {
+      browser.click('ul li:nth-child(3)');
+    });
+
+    Terra.should.matchScreenshot();
```

Comment:
lets take add a screenshot name to both of these `Terra.should.matchScreenshot();` so we can differentiate between the before and after click interaction.